src/core/ipfs/files.js

```diff
@@ -53,6 +53,7 @@ module.exports = function files (self) {
       pull(
         pull.values([hash]),
         pull.asyncMap(self._dagS.get.bind(self._dagS)),
+        pull.take(1),
```

Comment:
Why would we have something in 3 lines when we can just use one? It adds unnecessary complexity, memory overhead and superfluous ops.